Assets/VRTK/Scripts/Interactions/VRTK_InteractTouch.cs

```diff
@@ -191,18 +195,22 @@ public Collider[] ControllerColliders()
             return (controllerCollisionDetector.GetComponents<Collider>().Length > 0 ? controllerCollisionDetector.GetComponents<Collider>() : controllerCollisionDetector.GetComponentsInChildren<Collider>());
         }
 
-        protected virtual void Awake()
-        {
-            controllerEvents = GetComponent<VRTK_ControllerEvents>();
-            controllerActions = GetComponent<VRTK_ControllerActions>();
-            VRTK_PlayerObject.SetPlayerObject(gameObject, VRTK_PlayerObject.ObjectTypes.Controller);
-            destroyColliderOnDisable = false;
-            var controllerHand = VRTK_DeviceFinder.GetControllerHand(gameObject);
-            defaultColliderPrefab = Resources.Load(VRTK_SDK_Bridge.GetControllerDefaultColliderPath(controllerHand));
+        protected virtual void Awake() {
+            if (this.isActiveAndEnabled && Application.isPlaying) Debug.Log("Touch Awake");
+            if (controllerEvents == null) controllerEvents = GetComponent<VRTK_ControllerEvents>();
+            if (controllerEvents == null) throw new System.Exception("InteractTouch requires non-null VRTK_ControllerEvents field controllerEvents.");
+            if (controllerActions == null) controllerActions = GetComponent<VRTK_ControllerActions>();
+            if (controllerActions == null) throw new System.Exception("InteractTouch requires non-null VRTK_ControllerActions field controllerAction.");
+            VRTK_PlayerObject.SetPlayerObject(controllerEvents.ControllerAlias, VRTK_PlayerObject.ObjectTypes.Controller);
         }
 
         protected virtual void OnEnable()
         {
+            if (this.isActiveAndEnabled && Application.isPlaying) Debug.Log("Touch Enabled");
```

Comment:
Same as above: Looks like this is a left-over from debugging? Should be removed if it is.